src/js/visual/configurator.js

```diff
@@ -1,18 +0,0 @@
-if (
-    typeof require !== "undefined" &&
-    (typeof window === "undefined" ||
-        // eslint-disable-next-line camelcase
-        typeof __webpack_require__ !== "undefined" ||
-        (typeof navigator !== "undefined" && navigator.product === "ReactNative"))
-) {
-    // eslint-disable-next-line no-redeclare,no-var
-    var base = require("../base");
-    require("./configurator-prc");
-    // eslint-disable-next-line no-redeclare,no-var
-    var ripe = base.ripe;
-}
-
-ripe.Configurator = ripe.ConfiguratorPrc;
-
-ripe.Configurator.prototype = ripe.ConfiguratorPrc.prototype;
-ripe.Configurator.prototype.constructor = ripe.Configurator;

```

Comment:
I suppose this was a leftover file that needed to be deleted after the creation of `configurator-prc.js` right?

Reply:
Previously the configurator class was only a mere redirect to configuratorPRC, so in this PR it serves no purpose, therefore it could be safely removed

Reply:
Previously the configurator class was only a mere redirect to configuratorPRC, so in this PR it serves no purpose, therefore it could be safely removed